plaso/cli/pinfo_tool.py

```diff
@@ -494,7 +519,7 @@ def _PrintSessionsDetails(self, storage_reader):
     Args:
       storage_reader (BaseStore): storage.
     """
-    for session_number, session in enumerate(storage_reader.GetSessions()):
+    for _, session in enumerate(storage_reader.GetSessions()):
```

Comment:
Remove the enumerate here, it's not needed any more

Reply:
ack